inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -13,6 +13,9 @@ namespace vpu {
 
 namespace {
 
+// Current NMS implementation doesn't allow calculation of `> boxesTreshold` boxes using one SHAVE
+constexpr int boxesThreshold = 3650;
```

Comment:
Make it local to `getSHAVEsRequirementsImpl` or if's semantically the same as for `nms`, so put it in common place

Reply:
I've made `StaticShapeNMS` to be subclass `NonMaxSuppresion` so now there is only one copy of  these constant.